packages/rancher-logging/generated-changes/overlay/templates/loggings/aks/logging.yaml

```diff
@@ -15,6 +15,31 @@ spec:
     inputTail:
```

Comment:
If we are copying and pasting the exact same values, I think these should either belong in a helper function or we should define a `_logging.tpl` that we call for each of these logging.yaml files.

e.g. see the [`_pod.tpl`](https://github.com/grafana/helm-charts/blob/main/charts/grafana/templates/_pod.tpl) pattern in Grafana

Reply:
Filed as https://github.com/rancher/tasks/issues/191. Planned for this PR.